runtime/src/bank.rs

```diff
@@ -4295,6 +4297,15 @@ impl Bank {
         }
     }
 
+    // A cluster specific feature gate, when not activated it keeps v1.13 behavior in mainnet-beta;
+    // once activated for v1.14+, it allows compute_budget::request_heap_frame and
+    // compute_budget::set_compute_unit_price co-exist in same transaction.
+    pub(crate) fn enable_request_heap_frame_ix(&self) -> bool {
```

Comment:
nit: Think we don't need `pub(crate)`, don't see any callers outside of `bank.rs`.

That being said, if we move the function to `accounts.rs` and pass `feature_set` and `cluster_type()` as args, we could avoid having this logic in two places (there and here)

Reply:
right, initially thought to make it available to `accounts.rs`, but opt for simple dup code for now. Removing it.